ui/README.md

```diff
@@ -0,0 +1,23 @@
+# Kubecost Open Source UI
+
+## Installation & Running
+To run the UI, open a terminal to the `cost-model/ui/` directory (where this README is located) and run
+
+```
+npm install
+```
+
+This will install required depndencies and build tools. To launch the UI, run
+
+```
+npx parcel src/index.html
+```
+
+This will launch a development server, serving the UI at `http://localhost:1234` and targeting the data for an instance of
+Kubecost running at `http://localhost:9090`. To access an arbitrary Kubecost install, you can use
+
+```
+kubectl port-forward deployment/kubecost-cost-analyzer 9090
+```
+
```

Comment:
I assume the eventual goal is to package this up more nicely for an in-cluster deploy? Obviously no pressure to do that now, just wanted to know.

Reply:
Yep, that's what I'm thinking. In the future it can launch as a part of the cluster.